.github/workflows/frontend-release.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish frontend docker image on DockerHub
+
+on:
+  push:
+    tags:
+      - frontend/*
+
+jobs:
+  publish-docker-image:
+    runs-on: ubuntu-latest
+    environment: release
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          flavor: latest=false
+          images: concordium/ccdscan-frontend
+          tags: type=match,pattern=frontend/(.*),group=1
```

Comment:
We can add some checks that the image does not already exists on docker hub and a check that the tagged version matches the version in the `package.json` file:
```
      - name: Extract tag from package.json
        id: meta
        run: |
          export VERSION=$(jq -r .version simpleAgeVerification/package.json)
          export FULL_IMAGE_TAG="${{ env.REGISTRY }}/concordium/$IMAGE_NAME:$VERSION"
          echo "::notice FULL_IMAGE_TAG=${FULL_IMAGE_TAG}"
          # Make sure the image does not exist. Abort if we can retrieve any metadata.
          if docker manifest inspect ${FULL_IMAGE_TAG} > /dev/null; then
             echo "::error ${FULL_IMAGE_TAG} already exists"
             exit 1
          elif [ ! "${EXPECTED_TAG}" = "simple-age-verification/${VERSION}" ]; then
             echo "::error Expected tag ${EXPECTED_TAG} does not match the version ${VERSION}."
             exit 1
```

Reply:
I don't see a high risk of overwriting images, we would have to overwrite the git tag first and force push, and in this case it seems desirable to upload a new image again.

Reply:
I'll add the check for the package version